src/features/factory-tycoon/components/ResearchModal.tsx

```
@@ -1,134 +1,156 @@
+				{/* Header */}
+				<div className="flex items-center justify-between p-5 border-b border-[var(--factory-border)]">
+					<h2 className="text-xl font-bold text-[var(--factory-text-primary)] flex items-center gap-3">
+						<div className="w-10 h-10 rounded-lg bg-gradient-to-br from-purple-500 to-purple-700 flex items-center justify-center">
+							<Beaker className="w-5 h-5 text-white" />
+						</div>
+						Research Lab
+					</h2>
+					<button
+						onClick={onClose}
+						className="p-2 rounded-lg text-[var(--factory-text-muted)] hover:text-[var(--factory-text-primary)] hover:bg-white/5 transition-colors"
+					>
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)